textattack/commands/train_model/run_training.py

```diff
@@ -16,24 +17,151 @@
     augmenter_from_args,
     dataset_from_args,
     model_from_args,
+    attack_from_args,
     write_readme,
 )
 
 device = textattack.shared.utils.device
 logger = textattack.shared.logger
 
+def save_args(args, save_path):
+    # Save args to file
+    final_args_dict = {k: v for k, v in vars(args).items() if is_writable_type(v)}
+    with open(save_path, "w", encoding="utf-8") as f:
+        f.write(json.dumps(final_args_dict, indent=2) + "\n")
+
+def get_sample_count(*lsts):
+    if all(len(lst) == len(lsts[0]) for lst in lsts):
+        sample_count = len(lsts[0])
+    else:
+        sample_count = None
+    return sample_count
+
+def random_shuffle(*lsts):
+    permutation = np.random.permutation(len(lsts[0]))
+    shuffled = []
+    for lst in lsts:
+        shuffled.append((np.array(lst)[permutation]).tolist())
+    return tuple(shuffled)
+
+def train_val_split(*arrays, split_val=.2):
+    sample_count = get_sample_count(*arrays)
+    if not sample_count:
+        raise Exception("Batch Axis inconsistent. All input arrays must have first axis of equal length.")
+    arrays = random_shuffle(*arrays)
+    split_idx = math.floor(sample_count * split_val)
+    train_set = [array[split_idx:] for array in arrays]
+    val_set = [array[:split_idx] for array in arrays]
+    if len(train_set) == 1 and len(val_set) == 1:
+        train_set = train_set[0]
+        val_set = val_set[0]
+    return train_set, val_set
+
+def filter_labels(text, labels, allowed_labels):
```

Comment:
please add a docstring [sorry]

example of usage -- SNLI has -1 when the label is unclear. normal label set is 0,1,2. so users can specify --allowed-labels 0 1 2 and automatically skip train/eval samples with a -1 label